euphoria-examples/src/main/java/cz/seznam/euphoria/examples/wordcount/SimpleWordCount.java

```diff
@@ -172,9 +175,9 @@ private static Flow buildFlow(DataSource<String> input, DataSink<String> output)
     // defined function to these values. The result of this user defined
     // function is then emitted to the output along with its corresponding
     // key.
-    Dataset<Pair<String, Long>> counted = ReduceByKey.named("REDUCE")
+    final Dataset<Pair<String, Long>> counted = ReduceByKey.named("REDUCE")
         .of(words)
-        .keyBy(e -> e)
+        .keyBy(String::toLowerCase)
```

Comment:
Why do we lowercase the token here, in previous version it was not lowercase, do we want to change functionality here?